src/editor/Gui.cpp

```diff
@@ -2617,6 +2829,153 @@ void Gui::drawAbout() {
 	ImGui::End();
 }
 
+void Gui::drawMergeWindow() {

```

Comment:
leave out

Reply:
Now it works as from command line. "For non linux users" )))